src/handlers/bulk/shared/validator.js

```diff
@@ -203,9 +205,34 @@ const validateBulkTransferFulfilment = async (payload, headers) => {
     return { isValid, reasons }
   }
   isValid = isValid && await validateFspiopSourceAndDestination(payload, headers)
+  isValid = isValid && validateCompletedTimestamp(payload)
   return { isValid, reasons }
 }
 
+const validateCompletedTimestamp = (payload) => {
+  const maxLag = (Config.MAX_FULFIL_TIMEOUT_DURATION_SECONDS || defaultLagSeconds) * 1000
```

Comment:
Would it not be better to have the default values set in the config.js instead of here?